### ISSUE TITLE
OCI template default version

### ISSUE THREAD

**Issue by KaradzaJuraj** [cyclops-ui/cyclops#458]
When adding OCI charts as templates, if we do not enter the `version`, an un-descriptive error will pop up. This is because the OCI charts do not have a default fallback for version → currently, you **must** enter a valid version of the OCI chart.

We should implement a default that resolves to the latest released version.

**Steps to reproduce:**

1. go to `Templates` tab
2. click on `Add template reference`
3. try connecting an OCI chart, but do not input the version
    
    ```bash
    ### For example:
    repo: oci://registry-1.docker.io/bitnamicharts
    path: wordpress
    version:
    ```
    
4. **You will receive an error!**
5. But if you put `x.x.x` in the version field, the template will be successfully added

**Comment by pithecuse527** [comment]
Hello,
May I take this?

**Comment by KaradzaJuraj** [comment]
Hi @pithecuse527, of course you can!
I assigned you 🙂

**Comment by pithecuse527** [comment]
Here is the PR :)
https://github.com/cyclops-ui/cyclops/pull/461

**Comment by petar-cvit** [comment]
Resolved with #461
